src/buildMatchSnapshot.js

```diff
@@ -33,4 +42,19 @@ const buildMatchSnapshot = (utils, parseArgs) => {
   };
 };
 
+const safeRequireJestExpect = () => {
+  try {
+    return require("jest-matchers");
+  } catch (e) {
+    return null;
+  }
+};
+
+const JEST_MARKERS = ["enableAutomock", "genMockFromModule", "clearAllMocks", "runAllTicks"];
+
+const thisRunsInJest = () => (
+  typeof jest === "object" &&
+  JEST_MARKERS.every((marker) => typeof jest[marker] === "function")
```

Comment:
@cpojer is there a more canonical way to detect that we're running in jest? Or is this fine?